### ISSUE TITLE
Version 0.0.5 seems to incompatible with Request

### ISSUE THREAD

**Issue by ninozhang** [rc0x03/node-osmosis#12]
I use both Osmosis and Request in my project.
Request for fetching a task config from my server.
And Osmosis will do the crawling.
But after 5 ~ 6 times, Request will failed to response/callback correctly.
Plz have a look at it.

Code Like:
Request.get(url, function (err, res, body) {
  // after 5 ~ 6 times, this function will not be called any more, and just wait for the request timeout.
  osmosis.get(body).done();
});

**Comment by rc0x03** [comment]
This issue should be fixed in 0.0.6

Here's my test code:
```javascript
var osmosis = require('osmosis');
var request = require('request');

for (var i = 0; i < 10; i++) {
    request('http://google.com', function(err, res, body) {
        osmosis
        .parse(body)
        .set('title', 'title')
        .data(console.log)
    });
}
```
Also, is there a reason for not using Osmosis' built in request functionality?

**Comment by ninozhang** [comment]
In my project, I use Request to fetch a new task from my task center and do something.
Then get a url from the task and pass to Osmosis.
I didn't fetch a task by the Osmosis's built in request because I want to keep the Osmosis code clean.
